src/Microsoft.AspNetCore.DataProtection.AzureStorage/baseline.netcore.json

```diff
@@ -0,0 +1,156 @@
+{
+  "AssemblyIdentity": "Microsoft.AspNetCore.DataProtection.AzureStorage, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```

Comment:
Discussed offline&hellip; use whatever branch corresponds to the 1.0.0 version of this package.